### PR TITLE
docs: Update the Plasma Mobile docs

### PR DESCRIPTION
**Pull request by alistair23**
Signed-off-by: Alistair Francis <alistair.francis@wdc.com>

### CODE REVIEW

File: docs/Plasma-Mobile-on-Unleashed.md [hunk 1]
```diff
@@ -37,15 +37,19 @@ Follow the usual steps you use to build Yocto images. There are more details on
...
+INHERIT += "extrausers"
+EXTRA_USERS_PARAMS_freedom-u540 = "useradd kde; \
+                                   usermod -P kde kde;"
 
-PACKAGECONFIG_append_pn-mesa = "r600"
+## Use the Microsemi Expansion board DT in OpenSBI (passed to the kernel)
+RISCV_SBI_FDT_freedom-u540      = "HiFiveUnleashed-MicroSemi-Expansion.dtb"
+
+## Install GPU driver for the GPU being used, in this example it's a Radeon R5 230
+PACKAGECONFIG_append_pn-mesa    = "r600"
```
Comment (kraj) [inline docs/Plasma-Mobile-on-Unleashed.md]: missing space " r600" it should be

File: docs/Plasma-Mobile-on-Unleashed.md [hunk 2]
```diff
@@ -37,15 +37,19 @@ Follow the usual steps you use to build Yocto images. There are more details on
 Once you know you can build for the ```freedom-u540``` machine you should add these lines to your local.conf or distro:
 
 ```
-DISTRO_FEATURES_freedom-u540          += " wayland x11 opengl xattr gles directfb"
-DISTRO_FEATURES_freedom-u540          += " polkit kde pam pulseaudio"
-FEATURE_PACKAGES_append_freedom-u540   = " wayland mesa-megadriver"
-FEATURE_PACKAGES_append_freedom-u540   = " mesa-driver-swrast"
-IMAGE_INSTALL_append_freedom-u540      = " wayland pulseaudio pulseaudio-server"
+## Required DISTRO_FEATURES
+DISTRO_FEATURES_freedom-u540   += " wayland x11 kde opengl pam polkit xattr"
```
Comment (kraj) [inline docs/Plasma-Mobile-on-Unleashed.md]: this seems not correct, it should not be using MACHINE override

File: docs/Plasma-Mobile-on-Unleashed.md (hunk 1 above)
Comment (alistair23) [reply docs/Plasma-Mobile-on-Unleashed.md]: Ah, good catch.

File: docs/Plasma-Mobile-on-Unleashed.md (hunk 2 above)
Comment (alistair23) [reply docs/Plasma-Mobile-on-Unleashed.md]: As in remove the _freedom-u540?

Comment (kraj) [reply docs/Plasma-Mobile-on-Unleashed.md]: yes

**Comment by alistair23** [comment]
V2 pushed

File: docs/Plasma-Mobile-on-Unleashed.md [hunk 3]
```diff
@@ -59,7 +63,6 @@ Follow the top level README to flash the image to an SD card (you will probably
 
 ## Known issues
 
- * Qt5 base will most likely fail to cross compile for RISC-V. As a work around you can apply the patch from here: https://github.com/meta-qt5/meta-qt5/pull/156/commits
- * Sometimes the PCIe devices do not work on first boot. In you can't detect the PCIe devices on the board try power cycling the board and then they should appear.
+  * Sometimes the PCIe devices do not work on first boot. In you can't detect the PCIe devices on the board try power cycling the board and then they should appear.
```
Comment (gurjeet) [inline docs/Plasma-Mobile-on-Unleashed.md]: s/In you can't/In case you can't/

Comment (kraj) [reply docs/Plasma-Mobile-on-Unleashed.md]: thanks for pointing it out, can you propose a fix ? this has already been merged.

Comment (gurjeet) [reply docs/Plasma-Mobile-on-Unleashed.md]: Done. Please see https://github.com/riscv/meta-riscv/pull/130

**Pull request merged**